src/_guides/language/language-tour.md

```diff
@@ -983,66 +983,36 @@ For more information about maps, see
 [Generics](#generics) and
 [Maps](/guides/libraries/library-tour#maps).
 
-### Runes
+<a id="runes"></a>
+### Runes and grapheme clusters
 
 In Dart, runes are the UTF-32 code points of a string.
+As of Dart 2.6, instead of using runes use the
+[characters package,][characters package]
+which has better support for user-perceived characters,
+also known as
+[Unicode (extended) grapheme clusters.][grapheme clusters]
 
 Unicode defines a unique numeric value for each letter, digit,
 and symbol used in all of the world's writing systems.
 Because a Dart string is a sequence of UTF-16 code units,
 expressing 32-bit Unicode values within a string requires
 special syntax.
-
 The usual way to express a Unicode code point is
 `\uXXXX`, where XXXX is a 4-digit hexadecimal value.
 For example, the heart character (♥) is `\u2665`.
 To specify more or less than 4 hex digits,
 place the value in curly brackets.
 For example, the laughing emoji (😆) is `\u{1f600}`.
 
-The [String][]
-class has several properties you can use to extract rune information.
-The `codeUnitAt` and `codeUnit` properties return 16-bit code
-units. Use the `runes` property to get the runes of a string.
-
-The following example illustrates the relationship between runes, 16-bit code
-units, and 32-bit code points. Click **Run** to see runes in action.
-
-{% comment %}
-https://gist.github.com/589bc5c95318696cefe5
-{{site.dartpad}}/589bc5c95318696cefe5
-Unicode emoji: https://unicode.org/emoji/charts/full-emoji-list.html
-
-<?code-excerpt "misc/lib/language_tour/built_in_types.dart (runes)"?>
-```dart
-void main() {
-  var clapping = '\u{1f44f}';
-  print(clapping);
-  print(clapping.codeUnits);
-  print(clapping.runes.toList());
-
-  Runes input =
-      Runes('\u2665  \u{1f605}  \u{1f60e}  \u{1f47b}  \u{1f596}  \u{1f44d}');
-  print(String.fromCharCodes(input));
-}
-```
-{% endcomment %}
-
-<iframe
-src="{{site.dartpad-embed-inline}}?id=589bc5c95318696cefe5"
-    width="100%"
-    height="400px"
-    style="border: 1px solid #ccc;">
-</iframe>
+If you need to read or write individual Unicode characters,
+use the `Characters` class from the characters package.
+A `Characters` object is a string viewed as a sequence of grapheme clusters.
```

Comment:
Can we show some small example here, please? I think you can just grab part of the `example.dart` from the package.

Reply:
ptal

Reply:
Nice, thanks!